test/language/module-code/early-dup-export-star-as-dflt.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2018 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-module-semantics-static-semantics-early-errors
+description: >
+    It is a Syntax Error if the ExportedNames of ModuleItemList contains any
+    duplicate entries.
+flags: [module, export-star-as-namespace-from-module]
+negative:
+  phase: parse
+  type: SyntaxError
+---*/
+
+throw "Test262: This statement should not be evaluated.";
+
+var x;
+export default x;
+export * as default from 'early-dup-export-start-as-dflt.js';
```

Comment:
I think it would make sense to have a positive test case for `export * as default from`.